libs/extensions/angular/image-banner/src/image-banner.component.html

```diff
@@ -1,22 +1,27 @@
-<kirby-card *ngIf="externalLink" [themeColor]="backgroundBlur === 'none' ? 'white' : 'dark'">
-  <ng-container *ngTemplateOutlet="sharedCardContent"></ng-container>
-</kirby-card>
+@if (externalLink) {
+  <kirby-card [themeColor]="backgroundBlur === 'dark' ? 'dark' : 'white'">
```

Comment:
Wouldnt this read better? (please verify my logic 😅)
```suggestion
  <kirby-card [themeColor]="backgroundBlur === 'none' ? 'white' : backgroundBlur">
```